lightning/src/onion_message/functional_tests.rs

```diff
@@ -351,7 +351,13 @@ fn async_response_over_one_blinded_hop() {
 	let response_instruction = nodes[0].custom_message_handler.handle_custom_message(message, responder);
 
 	// 6. Simulate Alice asynchronously responding back to Bob with a response.
-	nodes[0].messenger.handle_onion_message_response(response_instruction);
+	let res = nodes[0].messenger.handle_onion_message_response(response_instruction);
+
+	match res {
+		Ok(Some(SendSuccess::Buffered)) => assert!(true),
+		_ => panic!("Unexpected event"),
```

Comment:
Event?

Reply:
Oops!
I meant to say something about the unexpected "state" of the result, but I confused it with the word "event"! 😅
Thanks for catching it!